onnxruntime/core/providers/nv_tensorrt_rtx/nv_execution_provider_custom_ops.cc

```diff
@@ -8,6 +8,12 @@
 #include "nv_execution_provider_custom_ops.h"
 #include "nv_execution_provider.h"
 
+#ifdef _WIN32
+#include <windows.h>
```

Comment:
Suggestion: abstract all the platform dependent things into a separate folder or at least a file.